packages/web/src/js/Tooltip.ts

```diff
@@ -267,6 +301,12 @@ class Tooltip extends BaseComponent {
     return null;
   }
 
+  getTrigger() {
+    const id = this.tip && (this.tip.getAttribute('id') as string);
+
+    return document.querySelector(`[data-spirit-target="#${id}"]`) as HTMLElement;
```

Comment:
```suggestion
    return document.querySelector<HTMLElement>(`[data-spirit-target="#${id}"]`);
```

Suggestions from yesterday:
- https://twitter.com/mattpocockuk/status/1756977158923182488?t=45mLaAXjSwMp4RECLro4qg&s=19
- https://github.com/total-typescript/total-typescript-book/blob/main/book-content/preview-01.md#not-all-functions-can-receive-types

Reply:
I see no change here.

Reply:
It will add the type 'HTMLElement | null,' and this will throw an error in our event handlers that are not expecting this value to be null. So i decided not to do this change.